cnf/ext/junit.bnd

```diff
@@ -1,5 +1,6 @@
 junit: org.apache.servicemix.bundles.junit;version="[4.11,5)",\
  org.assertj.core;version="[3,4)"
+junit-platform-reporting: 1.4.0
```

Comment:
I don't think we need this. We should just refer to `latest` version and control what version that is via the central.mvn file. Otherwise this will get out of sync eventually.

To the extent we need a version number to export the packages with a version, we should put that in the place where the export happens rather than here.

Reply:
Makes sense. Is there a macro to extract the version from a particular bundle/jar? This would be the easy way to export the right package version and avoid using another number that can get out-of-sync.

Reply:
I don't think we have a macro to get the value of a manifest header from a jar. But we could add one. However, for that jar, you would have to use something like `Implementation-Version` header.

Reply:
I tried using a combination of the repo macro and the last macro, thinking that repo would return a semicolon-separated list with the last element equal to the version of the package found in the repo. This caused a StackOverflowError in my Bndtools workspace... but I think the principle is sound? Is the version not stored as part of the gav coordinate (saves having to open the jar to find the header)?

Reply:
repo macro gives a jar file name and last returns the last item in a comma separated list of strings. Not sure how you get from a jar file name to a comma separated list of strings. With a replace macro, you could write a regex to extract the version number portion of the file name.

Reply:
> you could write a regex to extract the version number portion of the file name.

This should work on a jar from MavenBndRepository since the local file name will be in the local m2 repo and should have a version number in the jar file name.

Reply:
This point is now a bit moot as I have gone away from a separate bundle using the class files as-is and we are now working from (modified) source within `biz.aQute.tester.junit-platform`. However, for the record (as this technique might prove useful in the future), your suggestion worked - using the `replacestring` macro with the `repo` macro allowed me to extract the actual version string for `version=latest`. I managed to use this technique to automatically export the right package version (using the package number), but when I tried to use it for `Bundle-Version` I got a `StackOverflowError` (clearly the repo macro was causing some infinite recursion somewhere).